### ISSUE TITLE
spotBugs task complains about io.opentelemetry.instrumentation.auto.api.concurrent.State

### ISSUE THREAD

**Issue by imavroukakis** [open-telemetry/opentelemetry-java-instrumentation#1319]
While building locally via `./gradlew check` I get the following complaint from the spotBugs task

```
> Task :auto-api:spotbugsMain
H V MS: io.opentelemetry.instrumentation.auto.api.concurrent.State.FACTORY isn't final but should be  At State.java:[line 29]
SpotBugs reported failures
org.gradle.api.GradleException: Verification failed: SpotBugs violation found: 1
        at com.github.spotbugs.snom.internal.SpotBugsRunnerForWorker$SpotBugsExecutor.execute(SpotBugsRunnerForWorker.java:95)
```

Whilst, on the face of it, this is a valid complaint given the structure and usage of the class and a trivial fix,  I'm confused as to why this isn't caught in the CircleCI builds. Is it because this task is not running?

**Comment by trask** [comment]
hey @imavroukakis! ya we should probably fix if they seem reasonable, and then add `check` to CI to prevent regressions. see also #1166.

**Comment by imavroukakis** [comment]
OK, I raised https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/1325 to address the findings of `spotBugsMain` and `spotBugsTest` tasks.

**Comment by iNikem** [comment]
@imavroukakis is this now fixed by #1325?

**Comment by iNikem** [comment]
Closed by #1325

**Comment by imavroukakis** [comment]
> Closed by #1325

Yes, thanks for closing!
